lib/rules/web/admin_console/4.3/environments.xyaml

```diff
@@ -80,7 +80,7 @@ remove_env_vars:
       xpath: //div[contains(@class,'pairs-list__row')]
   - selector: &env_var_delete_icon |
       var parent_row = document.querySelectorAll('input[value=<env_var_name>]')[0].parentElement.parentElement
-      return parent_row.querySelector('.pairs-list__delete-icon')
+      return parent_row.querySelector("button[data-test-id='pairs-list__delete-btn']")
```

Comment:
We need click on the button not the icon since sometimes the icon can't be clicked